smoke-tests/wdio.CI.conf.js

```diff
@@ -0,0 +1,22 @@
+const { config } = require("./wdio.conf.js");
+
+(config.baseUrl = "http://localhost:3000"), // TODO:- Change to the url in the CI environment
```

Comment:
remind me to help you set this up

Reply:
Will do I'll ensure a ticket is created 👍